packages/core/src/shader/utils/generateUniformBufferSync.ts

```diff
@@ -0,0 +1,295 @@
+import type { Dict } from '@pixi/utils';
+import { mapSize } from '../utils';
+import { IUniformData } from '../Program';
+import { UniformGroup } from '../UniformGroup';
+import { uniformParsers } from './uniformParsers';
+import type { Renderer } from '../../Renderer';
+import type { Buffer } from '../../geometry/Buffer';
+
+export type UniformsSyncCallback = (...args: any[]) => void;
+
+const uboUpdate =  (_ud: any, _uv: any, _renderer: Renderer, _syncData: any, buffer: Buffer) =>
```

Comment:
We shouldn't use anonymous arrow functions when unnecessary. It obfuscates stack traces:

```suggestion
function uboUpdate(_ud: any, _uv: any, _renderer: Renderer, _syncData: any, buffer: Buffer)
```